cscs-checks/apps/cpmd/cpmd_check.py

```diff
@@ -5,60 +5,75 @@
 
 import reframe as rfm
 import reframe.utility.sanity as sn
+from hpctestlib.apps.cpmd import Cpmd
 
+REFERENCE_PERFORMANCE_SMALL = {
+    'dom:mc': {
+        'prod': (285.5, None, 0.20, 's')
+    },
+    'daint:mc': {
+        'prod': (285.5, None, 0.20, 's')
+    },
+}
 
-@rfm.parameterized_test(['small'], ['large'])
-class CPMDCheck(rfm.RunOnlyRegressionTest):
-    def __init__(self, scale):
-        self.descr = 'CPMD check (C4H6 metadynamics)'
-        self.maintainers = ['AJ', 'LM']
-        self.tags = {'production'}
+REFERENCE_PERFORMANCE_LARGE = {
+    'daint:mc': {
+        'prod': (245.0, None, 0.59, 's')
+    },
+}
 
-        self.valid_systems = ['daint:gpu']
-        if scale == 'small':
+
+@rfm.simple_test
+class CPMDCheck(Cpmd):
+    scale = parameter(['small', 'large'])
+    maintainers = ['AJ', 'LM']
+    tags = {'production'}
+    valid_systems = ['daint:gpu']
+    descr = 'CPMD check (C4H6 metadynamics)'
+    num_tasks_per_node = 1
+    valid_prog_environs = ['builtin']
+    modules = ['CPMD']
+    executable = 'cpmd.x'
+    input_file = 'ana_c4h6.in'
+    readonly_files = ['ana_c4h6.in', 'C_MT_BLYP', 'H_MT_BLYP']
```

Comment:
The input file information should be part of the base class, no?
And the name of the input file should be a parameter, similar to the Amber case.